sips/sips/sip-009-nft-standard.md

```diff
@@ -0,0 +1,85 @@
+## Preamble
+
+- Sip Number: 009
```

Comment:
Small style nitpicks:

* `SIP Number` <-- SIP should be all caps.
* `Created: 2020-12-10` should be `Created: 10 December 2020`.